endpoint/spec.md

```diff
@@ -600,13 +600,41 @@ TODO merge this into the previous 'protocol' section
   - If `protocoloptions.protocol` is a well-known protocol, the options MUST be
     compliant with the [protocol's options](#protocol-options).
 
+#### `messagegroups`
+
+The `messagegroups` attribute is an array of URI-references to message
+definition groups. The `messagegroups` attribute is used to reference
+message definition groups that are not inlined in the endpoint definition.
+
+It is STRONGLY RECOMMENDED that the `messageid` values of the referenced
+message definitions be unique across all message groups and all messages
+defined within the `messages` collection of this message group.
```

Comment:
Possibly: add: it's ok to reject update request if server chooses to validate that msgIDs are unique

on 9/26 call - backout this recommendation